filebeat/fileset/modules.go

```diff
@@ -322,14 +322,17 @@ func appendWithoutDuplicates(moduleConfigs []*ModuleConfig, modules []string) ([
 
 func (reg *ModuleRegistry) GetInputConfigs() ([]*common.Config, error) {
 	var result []*common.Config
-	for module, filesets := range reg.registry {
-		for name, fileset := range filesets {
-			fcfg, err := fileset.getInputConfig()
-			if err != nil {
-				return result, fmt.Errorf("error getting config for fileset %s/%s: %v",
-					module, name, err)
+	for _, modules := range reg.registry {
+		for module, filesets := range modules {
+			for name, fileset := range filesets {
+				fcfg, err := fileset.getInputConfig()
+				reg.log.Infof("%#v", fcfg)
```

Comment:
Is this a leftover log message?

Reply:
Yes it is. I'll remove that and do another search to make sure I didn't leave any others.